src/Learning/KNITransfer/CMakeLists.txt

```diff
@@ -4,4 +4,4 @@ add_executable(KNITransfer ${cppfiles})
 set_khiops_options(KNITransfer)
 
 target_link_libraries(KNITransfer PUBLIC KhiopsNativeInterface KWLearningProblem DTForest KMDRRuleLibrary
-                                         $<$<BOOL:${MPI}>:PLMPI> PLParallelTask)
```

Comment:
MPI is not necessary for KNITransfer ?

Reply:
Yes, I forgot this one, I have remove all these  `$<$<BOOL:${MPI}>:PLMPI>` to add only one in `PLParallelTask`. So `PLMPI` will be add transitively by any targets adding `PLParallelTask`.